faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -516,6 +565,35 @@ public function generateAlias($varValue, DataContainer $dc)
 		return $varValue;
 	}
 
+	/**
+	 * Return the SERP URL
+	 *
+	 * @param FaqModel $objFaq
+	 *
+	 * @return string
+	 */
+	public function getSerpUrl(FaqModel $objFaq)
+	{
+		/** @var FaqCategoryModel $objCategory */
+		$objCategory = $objFaq->getRelated('pid');
+		$jumpTo = (int) $objCategory->jumpTo;
```

Comment:
What if `$objCategory` is null?

Reply:
This shouldn't never happen in the backend view. In the sense of defensive programming I adjusted it e8307f0639aa9ecdd2eec8cf051307686415c1b3